src/pages/Home/index.js

```diff
@@ -252,6 +254,23 @@ class Home extends PureComponent {
           />
         ) : null}
         <GeneralWarning />
+        <Callout type="warning">
+          <div>
+            Do data resources managed by EMBL-EBI and our collaborators make a
+            difference to your work?
+            <br /> Please take 10 minutes to fill in our{' '}
+            <Link
+              href="https://www.surveymonkey.com/r/HJKYKTT?channel=[webpage]"
```

Comment:
I suspect the `channel` param will be used to track the resources users were visiting when opening the survey.

```suggestion
              href="https://www.surveymonkey.com/r/HJKYKTT?channel=InterPro"
```

Reply:
I initially thought the same, but in the document the link changes in other parts as `channel[email]`, `channel[twitter]`, etc. so I leave it as `[website]`

Reply:
I noticed that too, but I thought the links be updated, e.g.:

- link in an email: `channel=recipient@company.com`
- link in a X/Twitter post: `channel=@interprodb`
- link in a LinkedIn post: `channel=interpro-pfam`

I'll reach the people in charge of the survey and ask them to clarify.

Reply:
cool, let's wait for that...  but in the meantime, I implemented your suggestion about the link in the config. Please have a look to it

Reply:
Got confirmation that indeed we should leave it as it is.